docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -16,34 +16,38 @@ end box
 [-> LogicManager : execute("delete 1")
 activate LogicManager
 
-LogicManager -> AddressBookParser : parseCommand("delete 1")
-activate AddressBookParser
+LogicManager -> McGymmyParser : parse("delete 1")
+activate McGymmyParser
 
-create DeleteCommandParser
-AddressBookParser -> DeleteCommandParser
-activate DeleteCommandParser
-
-DeleteCommandParser --> AddressBookParser
-deactivate DeleteCommandParser
-
-AddressBookParser -> DeleteCommandParser : parse("1")
-activate DeleteCommandParser
+McGymmyParser -> PrimitiveCommandParser : parsePrimitiveCommand("delete", ["1"])
+activate PrimitiveCommandParser
 
 create DeleteCommand
-DeleteCommandParser -> DeleteCommand
+PrimitiveCommandParser -> DeleteCommand
 activate DeleteCommand
 
-DeleteCommand --> DeleteCommandParser : d
+DeleteCommand --> PrimitiveCommandParser : d
 deactivate DeleteCommand
 
-DeleteCommandParser --> AddressBookParser : d
-deactivate DeleteCommandParser
-'Hidden arrow to position the destroy marker below the end of the activation bar.
-DeleteCommandParser -[hidden]-> AddressBookParser
-destroy DeleteCommandParser
+|||
+
+PrimitiveCommandParser --> DeleteCommand : getParameterSet
+activate DeleteCommand
```

Comment:
Should this be -> instead of --> ?

Reply:
yup you're probably right. I mustve copy pasted wrongly somewhere